src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,53 @@
+import { Link } from "@canonical/react-components";
+import { useParams } from "react-router-dom";
+
+export type EntityDetailsRoute = {
+  userName: string;
+  modelName: string;
+  appName: string;
+};
+
+const Breadcrumb = (): JSX.Element => {
```

Comment:
Just to maintain consistency, we typically use the `function` signature for components and then export them when they are defined...

```ts
export default function Breadcrumb(): JSX.Element {
```

Reply:
Seems to be a mix in the codebase but no problem

Reply:
Hmm yeah we should fix that.